### ISSUE TITLE
TODO: DISTIRCTS - Model 3 - Random Forest

### ISSUE THREAD

**Issue by itisaasta** [ashwindeo/dataschoolers#12]

**Comment by ashwindeo** [comment]
This is done in https://github.com/ashwindeo/dataschoolers/blob/master/Classifier_Ensemble_Methods.ipynb

For the school districts dataset, it is prone to overfitting.
Accuracy on training data: 0.94
Accuracy on test data:     0.72
